src/Cli/dotnet/commands/dotnet-sdk/check/xlf/LocalizableStrings.cs.xlf

```diff
@@ -52,6 +52,11 @@
         <target state="translated">Rozhraní .NET {0} se už nepodporuje.</target>
         <note />
       </trans-unit>
+      <trans-unit id="RuntimeVersionCheckFailure">
+        <source>No version found to check if this is up to date. Please check for updates manually.</source>
+        <target state="needs-review-translation">Nebyla nalezena žádná verze, aby bylo možné zkontrolovat, zda je aktuální. Zkontrolujte prosím aktualizace ručně.</target>
+        <note>Translation might not be accurate as Poedit free 3.1.1 was used.</note>
```

Comment:
```suggestion
        <target state="needs-translation"></target>
        <note />
```
I do not think that the translation from poedit should be used yet as I feel that Microsoft's translator's might do a better job.